zenoh_router/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,16 @@
+name: Zenoh router
+files:
+- name: zenoh_router.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
```

Comment:
Consider adding the [http template here](https://github.com/DataDog/integrations-core/blob/3fe2b0b81d0888418f4b4c97f15b7332c82e288b/datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/http.yaml#L1).